components/usage/pkg/db/dbtest/user.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package dbtest
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gitpod-io/gitpod/usage/pkg/db"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+	"gorm.io/gorm"
+)
+
+type User struct {
+	ID           uuid.UUID      `gorm:"primary_key;column:id;type:char;size:36;"`
+	AvatarURL    string         `gorm:"column:avatarUrl;type:char;size:255;"`
+	Name         string         `gorm:"column:name;type:char;size:255;"`
+	FullName     string         `gorm:"column:fullName;type:char;size:255;"`
+	CreationDate db.VarcharTime `gorm:"column:creationDate;type:varchar;size:255;"`
+
+	// user has more field but we don't care here as they are just used in tests.
+}
```

Comment:
This should be in the `db` package, not the `dbtest` package

Reply:
I put it here, because we don't need this in production code, it's only used for setting up testing state in DB.

Reply:
I see. I guess I can live with this, but I'd generally define it on the `db` package as someone else will come along and not find the User object there so they would go and define it again